units/initrd-battery-check.service.in

```diff
@@ -12,10 +12,14 @@ Description=Check battery level during early boot
 Documentation=man:initrd-battery-check.service(8)
 DefaultDependencies=no
 AssertPathExists=/etc/initrd-release
-Before=local-fs-pre.target
+After=plymouth-start.service
+Before=local-fs-pre.target local-fs.target
```

Comment:
i think we should just order this before initrd-root-device.target, i.e. the milestone after which the root device will be set up and later mounted, and thus be written to.